tests/aggregation/tests.py

```diff
@@ -884,7 +884,7 @@ def test_order_of_precedence(self):
         self.assertEqual(p2, {'avg_price': Approximate(Decimal('53.39'), places=2)})
 
     def test_combine_different_types(self):
-        msg = 'Expression contains mixed types. You must set output_field.'
+        msg = 'Expression contains mixed types: FloatField, NoneType. You must set output_field.'
```

Comment:
Please see my [comment](https://github.com/django/django/pull/11457/files#r291970661). It shouldn't return `NoneType`. You cannot add `source = None` it doesn't fix anything because `source` will not be overwritten by:
```python
any(not isinstance(output_field, source.__class__) for source in sources_iter)
```

Reply:
Please check the changes now. Also, can you check the failure? Doesn't seem like it's from my side.